### PR TITLE
Condensing Clean and Clean_Blood

### PR DESCRIPTION
**Pull request by tichys**
No use in having 2 of the same procs

### CODE REVIEW

File: code/game/objects/effects/decals/cleanable.dm [hunk 1]
```diff
@@ -35,11 +35,11 @@
...
 
-/obj/effect/decal/cleanable/clean_blood(ignore = 0)
-	if(!ignore)
+/obj/effect/decal/cleanable/clean(clean_forensics = TRUE)
+	if(clean_forensics)
 		qdel(src)
-		return
-	..()
+		return TRUE
+	. = ..()
```
Comment (aidan-bowman) [inline code/game/objects/effects/decals/cleanable.dm]:
```suggestion
	return ..()
```

File: code/game/objects/effects/decals/Cleanable/humans.dm [hunk 2]
```diff
@@ -34,14 +34,14 @@ var/global/list/image/splatter_cache=list()
...
-		set_invisibility(100)
+/obj/effect/decal/cleanable/blood/clean(clean_forensics = TRUE)
+	fluorescent = FALSE
+	if(invisibility != INVISIBILITY_ABSTRACT)
+		set_invisibility(INVISIBILITY_ABSTRACT)
 		amount = 0
 		STOP_PROCESSING(SSobj, src)
 		remove_extension(src, /datum/extension/scent)
-	..(ignore=1)
+	. = ..(clean_forensics = FALSE)
```
Comment (aidan-bowman) [inline code/game/objects/effects/decals/Cleanable/humans.dm]:
```suggestion
	return ..(clean_forensics = FALSE)
```

File: code/_helpers/washing.dm [hunk 3]
```diff
@@ -1,89 +1,9 @@
 /proc/wash_mob(mob/living/washing)
-
 	if(!istype(washing))
 		return
-
 	var/mob/living/L = washing
-
 	if(L.on_fire)
-		L.visible_message(SPAN_DANGER("A cloud of steam rises up as the water hits \the [L]!"))
+		L.visible_message("<span class='danger'>A cloud of steam rises up as the water hits \the [L]!</span>")
```
Comment (aidan-bowman) [inline code/_helpers/washing.dm:6]: why remove the span macro??

Comment (aidan-bowman) [reply code/_helpers/washing.dm:6]: also this should probably be a balloon alert instead

File: code/_helpers/washing.dm [hunk 4]
```diff
@@ -1,89 +1,9 @@
 /proc/wash_mob(mob/living/washing)
-
 	if(!istype(washing))
 		return
-
 	var/mob/living/L = washing
-
```
Comment (aidan-bowman) [inline code/_helpers/washing.dm:7]: ??? why are you deleting spacing

File: code/_helpers/washing.dm (hunk 3 above)
Comment (Tsurupeta) [reply code/_helpers/washing.dm:6]: No it shouldnt

Comment (aidan-bowman) [reply code/_helpers/washing.dm:6]: > No it shouldnt

buh buh my TG feedback guidelines

**Review by Tsurupeta — APPROVED** [review]
ok nothing can totally go wrong

**Pull request merged**